pkg/kapp/app/recorded_app.go

```diff
@@ -415,6 +427,9 @@ func (a *RecordedApp) Rename(newName string, newNamespace string) error {
 func (a *RecordedApp) renameConfigMap(app *corev1.ConfigMap, name, ns string) error {
 	oldName := app.Name
 
+	// Use app label as app change label for new, renamed or migrated apps
```

Comment:
How about apps that have been migrated, but use app label name instead of ID right now. It is not super clear to me how we are handling those cases 🤔 
1. Will these only be renamed _if_ the user renames the app?
2. Does renaming the app to the same name rename CMs?